resources/js/components/Elements/Slider.vue

```diff
@@ -91,6 +88,9 @@
         },
         computed: {
             slider() {
+                if(!this.reference) {
+                    return this.$el;
```

Comment:
```suggestion
                    return this.$el.firstChild;
```

This way we don't need the reference by default at all right?

Reply:
This is very true, although I feel like to some degree it's just moving the problem laterally 😅

It at least made sense for it to use the root node of the component when not given a reference, but to me this is a little too specific. I guess "overfitting" is the way I'd describe it? Not sure if I'm into it.